### ISSUE TITLE
Button: Angular Support

### ISSUE THREAD

**Issue by camwest** [orion-ui/orion#34]
```
As a developer
I want to import an angular Button instead of a web components Button
So that I can avoid learning the web components API
```

The following code should work:

```javascript
import 'angular'
import button from 'orion-angular';

angular.module('app', ['button'])
  .controller('AppController', function() {
    var app = this;

    app.buttons = [
      {size: '1', label: 'Normal button'},
      {size: '2', label: 'Pill Button'},
      {size: '3', label: 'Thin Button'},
      {size: '4', label: 'Normal with icon' }
    ];

    app.handleClick = (button) => {
        alert('button clicked!');
    }
  });

```

```html
<html ng-app="app">
<body>
    <ul ng-controller="AppController as app">
      <li ng-repeat="button in app.buttons">
        <orion-button size="{{button.size}}" ng-click="app.handleClick(button)">
          {{button.label}}
        </orion-button>
      </li>
    </ul>
</body>
</html>
```

FYI I'm pretty flexible about how we handle module loading so feel free to suggest an alternative. We'll also need to set up an angular-playground to test this.

# Done

* [x] angular-playground shows button and props can change dynamically
* [x] Angular button is documented in storybook
* [x] angular-playground works in Firefox, Safari, Chrome, and Edge

**Comment by camwest** [comment]
@marclove can you add the custom-elements-polyfill to the Angular playground so that it works in Firefox and Safari? See react-playground/src/App.js for an example.

**Comment by camwest** [comment]
Fixed in #99
